lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -2119,7 +2119,7 @@ GLStatus TFGraphLowering::visitGraphOperationInst(GraphOperationInst *inst) {
 
   // Remember each of the results.
   for (unsigned i = 0; i != numOpResults; ++i) {
-    addValueMapping({inst->getResult(0), 0}, {result, (int)i});
+    addValueMapping({inst->getResult(i), 0}, {result, (int)i});
```

Comment:
nice!

after completing the migration to graph op inst, which natively supports multiple result values, my understanding is that we can remove the second component `0` in `{inst->getResult(i), 0}` -- is that correct?

Reply:
yes, exactly right!  I look forward to that day.